cluster/ci/monitoring/mixins/prometheus/ghproxy_alerts.libsonnet

```diff
@@ -0,0 +1,48 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'ghproxy',
+        rules: [
+          {
+            alert: 'ghproxy-status-code-abnormal-%sXX' % code_prefix,
+            // excludeing 304 because 304 is not expected.
+            // https://developer.github.com/v3/#conditional-requests
+            // excluding 404 because otherwise
+            // "this is going to spam us just about 24/7 with the OWNERS thing"
+            // TODO: Undo 404 after https://jira.coreos.com/browse/DPTP-447 is done
+            expr: |||
+              sum(github_request_duration_count{status=~"%s..", status!="304|404"})/sum(github_request_duration_count{status!="404"}) * 100 > 5
+            ||| % code_prefix,
+            'for': '1m',
+            labels: {
+              severity: 'slack',
+            },
+            annotations: {
+              message: 'ghproxy has {{ $value | humanize }}%% of status code %sXX for over 1 minute.' % code_prefix,
+            },
+          }
+          for code_prefix in ['3', '4', '5']
```

Comment:
I added 3 here because i want to include non-304 error in.
We can remove them if it cause false alerts too often.